.github/workflows/build_and_test_dbt.yaml

```diff
@@ -84,14 +84,14 @@ jobs:
           if [[ $MODIFIED_RESOURCES_ONLY == 'true' ]]; then
             if [[ $MANUALLY_DISPATCHED == 'true' ]]; then
               echo "Running tests on manually selected resources"
-              dbt test -t "$TARGET" -s ${{ inputs.models }} --exclude "tag:test_qc*" --defer --state "$STATE_DIR"
+              dbt test -t "$TARGET" -s ${{ inputs.models }} --exclude "tag:data_test_iasworld" --defer --state "$STATE_DIR"
```

Comment:
Note that I decided on `data_test_iasworld` instead of `test_data_iasworld` because I doubt we'll need to tag our unit tests in any specific way such that we want `test_*` to be a pattern that encompasses `test_data_*` and `test_unit_*`. In the absence of that need, `data_test_*` feels clearer to me, although that's a subjective choice. I don't feel super strongly about and would be happy to switch up the tag if you have a strong opinion.

Reply:
That works for me!